students/saleta_sebastian/lesson_06_dicts_tuples_sets_args_kwargs/cubes.py

```diff
@@ -0,0 +1,24 @@
+def print_set(cubes_set):
+    print(len(cubes_set))
+    print(*[str(item) for item in sorted(cubes_set)])
+
+
+def cubes():
+    alice_cube, bob_cube = [int(s) for s in input().split()]
+    alice_colors_set = {int(input()) for s in range(alice_cube)}
```

Comment:
Loop variable ```s``` variable is not used. In such cases instead of any variable name, you can use just ``_``` which means that you don't care about this variable